src/Fixer/Comment/HeaderCommentFixer.php

```diff
@@ -145,14 +146,20 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
      */
     protected function createConfigurationDefinition()
     {
+        $fixerName = $this->getName();
+
         return new FixerConfigurationResolver([
             (new FixerOptionBuilder('header', 'Proper header content.'))
                 ->setAllowedTypes(['string'])
-                ->setNormalizer(static function (Options $options, $value) {
+                ->setNormalizer(static function (Options $options, $value) use ($fixerName) {
                     if ('' === trim($value)) {
                         return '';
                     }
 
+                    if (false !== strpos($value, '*/')) {
+                        throw new InvalidFixerConfigurationException($fixerName, 'Cannot use \'*/\' in header.');
```

Comment:
Can't `$this->getName()` be instead of `$fixerName` here?

Reply:
I first thought the same, however, this is a static closure, thus not bound to `$this`. See http://php.net/manual/en/functions.anonymous.php#functions.anonymous-functions.static

Reply:
Do we need this closure to be static?